README.md

```diff
@@ -23,7 +24,12 @@
     echo '[ -f /opt/boxen/env.sh ] && source /opt/boxen/env.sh' >> ~/.zshrc
 
 It should run successfully, and should tell you to source a shell script
-in your environment.
+in your environment. If it does not run successfully, and gives you
+errors about unable to resolve `github.gds`, make sure you're on the
+Bardeen wireless network and can access `github.gds` in a web browser.
```

Comment:
Not just Bardeen - probably better to say "any of the internal staff wifi networks".